### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by AvineshTripathi**
fixed issue number #126 

 corrected some grammatical errors

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ Be sure to include as much information including screenshots, text output, and b
 
 ## How to Request Enhancements
 
-First, please refer to the applicable [GitHub repository][github-repo] and search [the repository's GitHub issues][issues-list] to make sure your idea has not been (or is not still) considered.
+Firstly please refer to the applicable GitHub repository[github-repo] and search the repository's GitHub issues[issues-list] to make sure that your idea has not been (or is not still) considered.
 
-Then, please [create a new issue in the GitHub repository][new-issue] describing your enhancement.
+If it is not yet considered or something more can be added then please create a new issue in the GitHub repository[new-issue] describing your enhancement.
```
Comment (Abhishek-kumar09) [inline CONTRIBUTING.md:17]: This is not looking good to me
image

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -145,7 +145,7 @@ Skip the yarn part and proceed to the Pushing the changes section [here](#-pushi
 
 ## 📌 Pushing your changes
 
-Now you have made the changes , tested them, and built them. so now its time to push them.
+Now you have made the changes , tested them, and built them so now its time to push them.
```
Comment (Abhishek-kumar09) [inline CONTRIBUTING.md:148]: Remove space before comma.
Change it to:
Once you made the changes, and tests are successfully passing you can push your changes to remote.

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]
Changes

**Review by AvineshTripathi — COMMENTED** [review]
I solved the problem with the hyperlink but couldn't find way to solve the space thing as if i delete the space it shows error

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -12,9 +12,9 @@ Be sure to include as much information including screenshots, text output, and b
 
 ## How to Request Enhancements
 
-First, please refer to the applicable [GitHub repository][github-repo] and search [the repository's GitHub issues][issues-list] to make sure your idea has not been (or is not still) considered.
+Firstly please refer to the applicable GitHub repository [github-repo] and search the repository's GitHub issues [issues-list] to make sure that your idea has not been (or is not still) considered.
```
Comment (Abhishek-kumar09) [inline CONTRIBUTING.md:15]: Github repository is mentioned twice and issues is also written twice.
You can revert these changes

File: CONTRIBUTING.md (hunk 2 above)
Comment (Abhishek-kumar09) [reply CONTRIBUTING.md:148]: Change this sentence to : `Once you made the changes, and tests are successfully passing you can push your changes to remote.`

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]

**Pull request merged**